client/my-sites/patterns/components/readymade-template-details/index.tsx

```diff
@@ -67,11 +53,31 @@ export const ReadymadeTemplateDetails: ReadymadeTemplateDetailsFC = ( { id, rend
 							<div className="readymade-template-details-preview-mobile">
 								{ renderPreview?.( readymadeTemplate ) }
 							</div>
-							<div
-								className="readymade-template-details-description"
-								// eslint-disable-next-line react/no-danger
-								dangerouslySetInnerHTML={ { __html: readymadeTemplate.description } }
-							/>
+							<div className="readymade-template-details-description">
+								<div // eslint-disable-next-line react/no-danger
+									dangerouslySetInnerHTML={ { __html: readymadeTemplate.description } }
+								/>
+								<h4>{ translate( 'Customize it with AI' ) }</h4>
+								<p>
+									{ translate(
+										'Start with this layout and use our AI assistant to create the website of your dreams without breaking a sweat.'
+									) }
+								</p>
+								<p>
+									{ translate(
+										'Just describe your site in a few sentences, and our AI tool will customize the content for you.'
+									) }
+								</p>
+								<h4>{ translate( 'Need full control?' ) }</h4>
+								<p>
+									{ translate(
+										'If you want even more control, our powerful site editing tools are always at your disposal, allowing you tu customize every single detail of this beautiful layout.'
+									) }
+								</p>
+								<a href={ localizeUrl( 'https://wordpress.com/support/site-editor/' ) }>
+									{ translate( 'Learn more about how the block editor works.' ) }
```

Comment:
should this say Site Editor? I could be completely wrong but I don't think we usually use the phrase "block editor" for user facing text.

Reply:
Yeah, that makes sense. Changed to "site editor"